swift/apple/FirezoneNetworkExtension/Info.plist

```diff
@@ -2,13 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>NSExtension</key>
```

Comment:
Redundant and generates a warning about the SystemExtension being in the wrong directory.